### ISSUE TITLE
Database name contains random prefix - can't setup it on DB container

### ISSUE THREAD

**Issue by keritaf** [bitnami/bitnami-docker-testlink#55]
I'm trying to setup a testlink instance with docker-compose. I'm using mariadb:latest image with `bitnami_testlink` db pre-created. When I start the containers, testlink container restarts with this log (header removed):

```
nami    INFO  Initializing testlink
mysql-c INFO  Trying to connect to MySQL server
mysql-c INFO  Found MySQL server listening at db:3306
mysql-c INFO  MySQL server listening and working at db:3306
Error executing 'postInstallation': MySQL command failed to run. Error:
 ERROR 1044 (42000) at line 1: Access denied for user 'testlink'@'%' to database 'bitnami_testlink_1WKYf'
```

Notice the database name - it is different on every restart. It seems that database name includes random postfix - and as a result I can't precreate db and grant user rights to it.

Is there a way I can specify/fix database name used?

**Comment by juan131** [comment]
Hello @keritaf 

At this moment that behaviour is only supported on the Bitnami Docker Image for WordPress. I mean, you cannot pre-create a database for TestLink since it will auto-detect that there's a database already created with the default name (bitnami_testlink) and, instead of considering that's the one to use, the logic assumes it's necessary to create a new database so it tries to create a database with name `bitnami_testlink_RANDOM` (please note that RANDOM will be substituted with a random string).

In other words, you don't need to pre create anything. TestLink will create everything it needs on MariaDB.

**Comment by juris** [comment]
@keritaf If you want to use pre-created database. E.g. when you restore from backup.
Try placing empty files `.initialized` and `.restored` in `appdata` and `apache` directories.

**Comment by juan131** [comment]
Hi @juris have you tried that before? I don't think that would work since when initialising the container, it would not configure TestLink nor Apache properly since it will consider them as already configured.

**Comment by juris** [comment]
@juan131 I do it all the time when restoring from backups.
